week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -25,6 +35,14 @@ function exerciseOne(arrayOfPeople) {
  *
  */
 function exerciseTwo(shopping) {
+  var ul = document.createElement("ul");
+  content.appendChild(ul);
+  shopping.map(function(item){
+  let li = document.createElement("li");
+  ul.appendChild(li);
+  li.innerHTML = item;
+  })
```

Comment:
Some small formatting errors here. Could you use something like this?

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
Done. only one of my computers did not have prettier now fixed